task/github-set-status/0.2/README.md

```diff
@@ -44,12 +44,14 @@ kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/gi
   from the status of other systems. _e.g:_ `continuous-integration/tekton`
 * **STATE**: The state of the status. Can be one of the following `error`,
   `failure`, `pending`, or `success`.
+* **AUTH_TYPE**: The type of authentication to use. You could use the less secure "Basic"
+      for example
```

Comment:
It would be nice to have a list of valid values or link to a doc with that list

Reply:
Link added!